src/cassandra/statement.rs

```diff
@@ -283,6 +285,14 @@ impl Statement {
         }
     }
 
+    /// Sets the statement's timeout for waiting for a response from a node.
```

Comment:
The C++ docs say

> Request timeout in milliseconds. Use 0 for no timeout or CASS_UINT64_MAX to disable (to use the cluster-level request timeout).

Requiring a specific large number is messy, so I think we should model disabling by `None`. That is, have this method take `Option<Duration>`, where `Some(Duration::from_millis(0))` indicates no timeout, and `None` indicates disable (to use the cluster-level request timeout).

Reply:
Okay, so does that mean we want something like:
```
match timeout {
None => cass_statement_set_request_timeout(self.0, CASS_UINT64_MAX),
Some(time) => cass_statement_set_request_timeout(self.0, time.num_milliseconds() as u64),
}
```
* I don't think we would need a separate branch for the no timeout case, right?
* Is that the correct way to disable?

Reply:
Yes, correct in both cases. I would write it without repetition, though:
```rust
let timeout_millis = match timeout {
    None => CASS_UINT64_MAX,
    Some(time) => time.num_miliseconds(),
};
cass_statement_set_request_timeout(self.0, timeout_millis)
```

Reply:
Thanks, changed this now.

Reply:
Please can you add to the comment, so people don't have to refer to the C++ docs. I think it needs to say
> `Some(Duration::from_millis(0))` sets no timeout, and `None` disables it (to use the cluster-level request timeout).

Reply:
Yep, good point. Added.